CHANGELOG.md

```diff
@@ -19,6 +19,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
       of returning `none` when the operation did not return output.
   - **CUMULUS-2309**
     - Removed `@cumulus/api/models/granule.unpublishAndDeleteGranule` in favor of `@cumulus/api/lib/granule-remove-from-cmr.unpublishGranule` and `@cumulus/api/lib/granule-delete.deleteGranuleAndFiles`.
+  - **CUMULUS-2385**
+    - Update `sf-event-sqs-to-db-records` to write a granule's files to Postgres
```

Comment:
This is a super nit but I _think_ to be consistent, we should write this as:
```suggestion
    - Updated `sf-event-sqs-to-db-records` to write a granule's files to Postgres
```

Reply:
Ah yeah, sure thing. Also updated `Postgres` to `PostgreSQL` for consistency.